testing/integration/ess/inspect_test.go

```diff
@@ -74,6 +74,9 @@ func TestInspect(t *testing.T) {
 			}})
 	require.NoErrorf(t, err, "Error when installing agent, output: %s", out)
 	check.ConnectedToFleet(ctx, t, fixture, 5*time.Minute)
+	assert.Eventually(t, func() bool {
```

Comment:
Any reason to not use `require.Eventually` here?  I believe test execution would continue even if the `assert.Eventually` fails?